### PR TITLE
Fetch bounties list and detail from litentry graph

### LINKED ISSUE #844: Bounties

### PR DESCRIPTION
**Pull request by ishrak-imam**
### Description

Fixes: https://github.com/litentry/litentry-app/issues/844

TODO:
- [ ] Add cache redirect when going from list to details view

### CODE REVIEW

File: src/api/hooks/useBounties.ts [hunk 1]
```diff
@@ -1,83 +1,82 @@
...
-    updateDue: undefined,
-  };
-
-  if (status.isCuratorProposed) {
-    result = {
-      ...result,
-      status: 'CuratorProposed',
-      curator: status.asCuratorProposed.curator,
-    };
+export const ACCOUNT_FIELDS = gql`
```
Comment (silva-fj) [inline src/api/hooks/useBounties.ts:6]: Let's create an issue to organize common queries or fragments so we can easily find them to reuse

File: src/context/LitentryApiContext.tsx [hunk 2]
```diff
@@ -26,6 +26,22 @@ export function LitentryApiClientProvider({children}: {children: React.ReactNode
   useEffect(() => {
     const init = async () => {
       const cache = new InMemoryCache();
+      // {
+      //   typePolicies: {
```
Comment (silva-fj) [inline src/context/LitentryApiContext.tsx]: is this a TODO? if so, let's create an issue otherwise let's clean this up

**Review by silva-fj — APPROVED** [review]

File: src/api/hooks/useBounties.ts (hunk 1 above)
Comment (ishrak-imam) [reply src/api/hooks/useBounties.ts:6]: Done!

File: src/context/LitentryApiContext.tsx (hunk 2 above)
Comment (ishrak-imam) [reply src/context/LitentryApiContext.tsx]: Done!

**Pull request merged**